db/database-config.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
This structure is used by `sequelize-ci` to determine environments when running migrations. For now it just lets us clean up the config a little bit.